src/pages/Quote.js

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from 'react';
+
+const API_URL = 'http://api.quotable.io/random';
+export default function Quote() {
+  const [quote, setQuote] = useState('');
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      const response = await fetch(API_URL);
+      const randomQuote = await response.json();
+      setQuote(randomQuote.content);
+    };
+    setTimeout(() => fetchQuote(), 60000);
+  });
+  return (
+    <h4>{ JSON.stringify(quote) }</h4>
+  );
+}
```

Comment:
- [ ] Nice work on the Quotes but you can make the content bigger and and use some image, color to make it more better.